<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>615</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Gonderici</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>451</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_receiver"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_message">
       <property name="cursor">
        <cursorShape>SizeFDiagCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Mesaj覺n覺z覺 yaz覺n...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_sendMessage">
         <property name="text">
          <string>Send Message</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sendAttachment">
         <property name="text">
          <string>Send Attachment</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
